packages/fx-core/src/component/generator/generatorAction.ts

```diff
@@ -143,16 +136,35 @@ export const unzipAction: GeneratorAction = {
     if (!context.zip) {
       throw new MissKeyError("zip");
     }
-    context.outputs = await unzip(
-      context.zip,
-      context.destination,
-      context.fileNameReplaceFn,
-      context.fileDataReplaceFn,
-      context.filterFn
-    );
+    const tryLimits = 2;
+    for (let i = 0; i < tryLimits; i++) {
```

Comment:
I would prefer to add a new action that checks the `context.outputs` and fetches the fallback if the output is empty, instead of modifying the `unzipAction`. We can discuss this further offline.